base/daemon/BaseDaemon.cpp

```diff
@@ -222,7 +225,11 @@ class SignalListener : public Poco::Runnable
 
                 /// This allows to receive more signals if failure happens inside onFault function.
                 /// Example: segfault while symbolizing stack trace.
+#if __cplusplus <= 201703L
```

Comment:
Need to enable C++20 in Arcadia.
The C++ committee does not have anything against C++20 (they didn't answer our email).